commands/version/README.md

```diff
@@ -0,0 +1,252 @@
+# `@lerna/version`
+
+> Bump version of packages changed since the last release
+
+## Usage
+
+```sh
+lerna version 1.0.1 # explicit
+lerna version patch # semver keyword
+lerna version       # select from prompt(s)
+```
+
+When run, this command does the following:
+
+1. Identifies packages that have been updated since the previous tagged release.
+2. Prompts for a new version.
+3. Modifies package metadata to reflect new release.
+4. Commits those changes and tags the commit.
+5. Pushes to the git remote.
+
+## Positionals
+
+### semver `bump`
+
+```sh
+lerna version [major | minor | patch | premajor | preminor | prepatch | prerelease]
+# uses the next semantic version(s) value and this skips `Select a new version for...` prompt
+```
+
+When run with this flag, `lerna version` will skip the version selection prompt and [increment](https://github.com/npm/node-semver#functions) the version by that keyword.
+You must still use the `--yes` flag to avoid all prompts.
+
+#### "Graduating" prereleases
+
+If you have any packages with a prerelease version number (e.g. `2.0.0-beta.3`) and you run `lerna version` with and a non-prerelease bump (`major`, `minor`, or `patch`), it will publish those previously pre-released packages _as well as_ the packages that have changed since the last release.
+
+## Options
+
+* [`--allow-branch`](#--allow-branch-glob)
+* [`--amend`](#--amend)
+* [`--commit-hooks`](#--commit-hooks)
+* [`--conventional-commits`](#--conventional-commits)
+* [`--changelog-preset`](#--changelog-preset)
+* [`--exact`](#--exact)
+* [`--force-publish`](#--force-publish)
+* [`--git-remote`](#--git-remote-name)
+* [`--git-tag-version`](#--git-tag-version)
+* [`--message`](#--message-msg)
+* [`--preid`](#--preid)
+* [`--push`](#--push)
+* [`--sign-git-commit`](#--sign-git-commit)
+* [`--sign-git-tag`](#--sign-git-tag)
+
+### `--allow-branch <glob>`
+
+A whitelist of globs that match git branches where `lerna version` is enabled.
+It is easiest (and recommended) to configure in `lerna.json`, but it is possible to pass as a CLI option as well.
+
+```json
+{
+  "command": {
+    "publish": {
```

Comment:
Is this correct @evocateur ? Do you configure `lerna version` by configuring publish on `lerna.json` ? Seems confusing